pom.xml

```diff
@@ -42,6 +42,30 @@
             <version>1.3</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
There are two versions for the junit-jupiter-api?

Reply:
I added additional dependancies while trying out various methods